packages/gatsby/src/utils/cpu-core-count.js

```diff
@@ -4,7 +4,7 @@
  * @returns {number} Count of the requested type of CPU cores. Defaults to number of physical cores or 1
  */
 
-const cpuCoreCount = (useEnvVar = false) => {
+const cpuCoreCount = (useEnvVar = true) => {
```

Comment:
@DSchau FWIW is `useEnvVar` ever passed in as `false`? I only quickly looked at the `git blame` for when this code was introduced, but AFAICT it was only sharp that was calling this with `cpuCoreCount()`, and all the other callers were `cpuCoreCount(true)`.

Now that `cpuCoreCount()` is effectively `cpuCoreCount(true)`, I'm thinking its simpler to just remove the `useEnvVar` parameter all together.

Reply:
It's not - but this reserves the right to do so if needed.

Reply:
I personally find such "this _is_ configurable ... but oh is never actually used" patterns in code confusing b/c I get stuck wondering "...so when is this used?" only to realize ~X minutes of wasted time later that it's not, so would personally excise the extra/unused configuration, but your call. :-)

Reply:
I don't disagree :) PRs welcome!